src/picongpu/include/plugins/SliceFieldPrinterMulti.tpp

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013-2014 Heiko Burau, Rene Widera, Felix Schmitt,
+ * Copyright 2013 Heiko Burau, Rene Widera, Felix Schmitt,
```

Comment:
removing something from the copyright line is not correct. pls update to *2013-2015* in this file